### PR TITLE
W-16203550 Update policies-included-jwt-validation.adoc

### PR DESCRIPTION
**Pull request by glenn-rodgers-sf**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

**Review by aneisen-mulesoft — APPROVED** [review]

File: gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc [hunk 1]
```diff
@@ -53,7 +53,7 @@ In Local Mode, you apply the JWT Validation policy to your API via declarative c
 | `signingKeyLength`
 | Optional
 | 256
-| Number indicating key length
+| Number indicating key length. Supported values are 256, 384, 512. For ES method, Flex Gateway only supports 256 and 384 key lengths.
```
Comment (Crispy-Salesforce) [inline gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc]:
```suggestion
| Number indicating key length. Supported values are 256, 384, 512. For ES method, Flex Gateway supports 256 and 384 key lengths.
```

File: gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc [hunk 2]
```diff
@@ -73,17 +73,17 @@ In Local Mode, you apply the JWT Validation policy to your API via declarative c
 | `jwksUrl`
 | Optional
 | `http://your-jwks-service.example:80/base/path`
-|
+| The JWKS server URLs that contain the public keys for the signature validation. Configure multiple JWKS servers with a coma-separated list of the URLs.
```
Comment (Crispy-Salesforce) [inline gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc]:
```suggestion
| JWKS server URLs that contain the public keys for the signature validation. Configure multiple JWKS servers with a comma-separated list of the URLs.
```

Comment (Crispy-Salesforce) [reply gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc]: (Globally) Choose to start all sentences with the "the" article or remove it.

File: gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc [hunk 3]
```diff
@@ -73,17 +73,17 @@ In Local Mode, you apply the JWT Validation policy to your API via declarative c
...
 | Optional
 | 60
-|
+| Amount of time, in minutes, during which the policy considers the JWKS valid.
 
 | `jwksServiceConnectionTimeout`
 | Optional
 | 10000
-|
+| Sets the maximum time, in milliseconds, to wait for a response from each JWKS service when authenticating the access token validation endpoint.
```
Comment (Crispy-Salesforce) [inline gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc]:
```suggestion
| Maximum time, in milliseconds, to wait for a response from each JWKS service when authenticating the access token validation endpoint.
```

Comment (Crispy-Salesforce) [reply gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc]: (Globally) Choose to start the sentences with a noun or with a verb

File: gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc [hunk 4]
```diff
@@ -194,15 +194,15 @@ Ignore this field if you selected *none* as JWT Signing Method.
 | JWKS URL
 | This field appears if you selected the *JWKS* method as JWT Key Origin. +
 Ignore this field if you selected *none* as JWT Signing Method.
-| The URL to the JWKS server.
+| The JWKS server URLs that contain the public keys for the signature validation. Configure multiple JWKS servers with a coma-separated list of the URLs.
```
Comment (Crispy-Salesforce) [inline gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc]:
```suggestion
| JWKS server URLs that contain the public keys for the signature validation. Configure multiple JWKS servers with a comma-separated list of the URLs.
```

File: gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc [hunk 5]
```diff
@@ -194,15 +194,15 @@ Ignore this field if you selected *none* as JWT Signing Method.
...
 
 | JWKS Caching Time To Live
-| The amount of time, in minutes, that the JKWS is valid. When the JKWS expires, the policy retrieves it again. The devault value is 60 minutes. +
+| The amount of time, in minutes, that the JWKS is valid. When the JWKS expires, the policy retrieves the JWKS from each JWKS servers again. The default value is 60 minutes. +
```
Comment (Crispy-Salesforce) [inline gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc]:
```suggestion
|Amount of time, in minutes, that the JWKS is valid. When the JWKS expires, the policy retrieves the JWKS from each JWKS server again. The default value is 60 minutes. +
```

File: gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc [hunk 6]
```diff
@@ -194,15 +194,15 @@ Ignore this field if you selected *none* as JWT Signing Method.
...
 Ignore this field if you selected *none* as JWT Signing Method.
 | This field input is the amount of time, in minutes, during which the policy considers the JWKS valid.
 
 | JWKS Service connection timeout (milliseconds)
-| Sets the maximum time, in milliseconds, to wait for a response when authenticating the access token validation endpoint. +
+| Sets the maximum time, in milliseconds, to wait for a response from each JWKS service when authenticating the access token validation endpoint. +
```
Comment (Crispy-Salesforce) [inline gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc]:
```suggestion
| Maximum time, in milliseconds, to wait for a response from each JWKS service when authenticating the access token validation endpoint. +
```

File: gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc [hunk 7]
```diff
@@ -311,7 +311,7 @@ The JWT is also propagated downstream, and you can access it by using the follow
 
 === Retry Mechanism
 
-If you selected JSON Web Key Set (JWKS) as JWT Key Origin, the policy uses a retry mechanism to handle failed JWKS fetches. Every time the cached JWKS expires or is absent and a new request is initiated, a request is sent to the JWKS service. If the request fails, a sequence of asynchronous requests with an exponential backoff delay is sent to the JWKS service until a new JWKS is obtained and cached again.
+If you selected JSON Web Key Set (JWKS) as JWT Key Origin, the policy uses a retry mechanism to handle failed JWKS fetches. Every time the cached JWKS expires or is absent and a new request is initiated, a request is sent to each JWKS service. If a request fails, a sequence of asynchronous requests with an exponential backoff delay is sent to the failed JWKS service until a new JWKS is obtained and cached again.
```
Comment (Crispy-Salesforce) [inline gateway/1.8/modules/ROOT/pages/policies-included-jwt-validation.adoc]:
```suggestion
If you selected JSON Web Key Set (JWKS) as JWT Key Origin, the policy uses a retry mechanism to handle failed JWKS fetches. Every time the cached JWKS expires or is absent and a new request is triggered, a request is sent to each JWKS service. If a request fails, a sequence of asynchronous requests with an exponential backoff delay is sent to the failed JWKS service until a new JWKS is obtained and cached again.
```

**Review by Crispy-Salesforce — APPROVED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**